### DISCUSSION TITLE
Generating Conformers from MOL/XYZ Files and Aligning Common Atoms in RDKit

### DISCUSSION THREAD

**Discussion by Itayitp** [rdkit/rdkit#8062] [Q&A]
1. Is it possible to generate conformers directly from MOL or XYZ files using RDKit?
I’ve tried several times but have not been successful in generating conformers from these file formats.

2. I have a set of 14 molecules with the same skeleton, and I would like to align them so that the positions of the common atoms are consistent across all of the XYZ files. Is there a command or script in RDKit to align the common atoms to the same positions?

**Comment by nonsensejoke** [comment]
you may consider to use xyz2mol to generate the topology (bonding info) first
